packages/core/src/sdk/analytics/index.tsx

```diff
@@ -1,4 +1,4 @@
-import type { AnalyticsEvent } from '@faststore/sdk'
+import type { UnknownEvent } from '@faststore/sdk'
```

Comment:
Along with another suggestion below
```suggestion
import type { AnalyticsEvent, UnknownEvent } from '@faststore/sdk'
```